app/static/src/app/plot.ts

```diff
@@ -13,13 +26,36 @@ export function filterSeriesSet(s: OdinSeriesSet, name: string): OdinSeriesSet {
     };
 }
 
-export function odinToPlotly(s: OdinSeriesSet, palette: Palette): WodinPlotData {
+export interface PlotlyOptions {
+    includeLegendGroup: boolean,
+    lineWidth: number
+    showLegend: boolean
+}
+
+const defaultPlotlyOptions = {
+    includeLegendGroup: false,
+    lineWidth: 2,
+    showLegend: true
+};
+
+export function odinToPlotly(s: OdinSeriesSet, palette: Palette, options: Partial<PlotlyOptions> = {}): WodinPlotData {
+    const plotlyOptions = {
+        ...defaultPlotlyOptions,
+        ...options
+    };
+
     return s.y.map(
         (el: number[], i: number): Partial<PlotData> => ({
-            line: { color: palette[s.names[i]] },
+            mode: "lines",
```

Comment:
plotly default is to show markers if less than 15 data points, but we want to show lines in all plots (data points are plotted separately)